backend/src/test/java/ai/verta/modeldb/ProjectTest.java

```diff
@@ -1262,6 +1576,10 @@ public void k_getProjectByName() {
           createProjectRequest.getName(),
           project.getName());
 
+      if (isRunningIsolated()) {
+        mockGetResourcesForAllEntity(Map.of(project.getId(), project), testUser2);
```

Comment:
should this method be "mockGetResourcesForAllEntities"?

Reply:
Done